### PR TITLE
State based stying for Media players & media loading status

### PR DESCRIPTION
**Pull request by dblodorn**
Added an additional prop to the getStyles function for additional selectors.

### CODE REVIEW

**Comment by dblodorn** [comment]
@iainnash this all is ready for review - also included the flag to remove default styling in the style object.

**Review by iainnash — CHANGES_REQUESTED** [review]
I like this approach overall! One concern is it’s a little fancy and wonder if writing out the entire classes is more explicit. Can go to either way. 

Main note is I believe aria-pressed can be a boolean in react and it gets passed correctly to the dom by the react engine.

**Comment by dblodorn** [comment]
With this PR we now have full classes being applied - so when play button is activated the class "zora-mediaPlayButton--playing" is added.

Screen Shot 2021-09-22 at 1 53 25 PM

**Comment by dblodorn** [comment]
If this is all good can we merge to master and publish a prerelease?

File: src/content-components/Audio.tsx [hunk 1]
```diff
@@ -155,6 +155,7 @@ export const AudioRenderer = forwardRef<HTMLAudioElement, RenderComponentType>(
             <Fragment>
               <button
                 aria-live="polite"
+                aria-pressed={playing ? true : false}
```
Comment (iainnash) [inline src/content-components/Audio.tsx:158]: nit: can just be playing

**Comment by iainnash** [comment]
Looks good!

**Pull request merged**